pkg/master/master.go

```diff
@@ -157,6 +146,13 @@ type Master struct {
 	serviceReadWritePort int
 	masterServices       *util.Runner
 
+	storage map[string]apiserver.RESTStorage
+
+	nodeRegistry      minion.Registry
```

Comment:
Hmm. Ok. I see their other uses. I suppose this is better than a map of registries in that it let's us know which are actually used.